### DISCUSSION TITLE
I'm new to this and I'm having trouble implementing the calender

### DISCUSSION THREAD

**Discussion by Jenin82** [adobe/react-spectrum#5547] [Q&A] [answered]
this is how the calender is rendering
![image](https://github.com/adobe/react-spectrum/assets/84625875/1e571114-1b85-49c6-811d-61cd3bcb1079)

```
import React, { useState } from 'react';
import { Calendar } from '@adobe/react-spectrum';
import { CalendarDate } from '@internationalized/date';

function MyCalendarComponent() {
  const [selectedDate, setSelectedDate] = useState(new CalendarDate(2023, 1, 1));

  return (
    <div>
      <Calendar 
        value={selectedDate} 
        onChange={setSelectedDate} 
      />
    </div>
  );
}

export default MyCalendarComponent;
```

**Comment by reidbarber** [comment, marked as answer]
When using React Spectrum components (from `@adobe/react-spectrum`), you'll need to wrap your app in a Provider, which provides the styles: https://react-spectrum.adobe.com/react-spectrum/getting-started.html#setting-up-your-app

**Reply by Jenin82** [reply]
Thanks you
It solved the issue

Is there any way to add custom style to the calendar ?

**Reply by yihuiliao** [reply]
In general, we don't allow custom styling to Spectrum components since they follow the Spectrum design system. If you have to, you can use UNSAFE_style but we discourage it. Instead, maybe you'd be more interested in using our RAC components which are unstyled but are built on top of the same hooks as React Spectrum components. Here you can find the docs for RAC Calendar: https://react-spectrum.adobe.com/react-aria/Calendar.html#calendar. 

You can also read more about what RAC is here: https://react-spectrum.adobe.com/react-aria/react-aria-components.html

**Reply by Jenin82** [reply]
This really helped. I'll be using both RAC and Spectrum on my future projects
